oci/repositories.bzl

```diff
@@ -107,12 +107,13 @@ def oci_register_toolchains(name, register = True):
         register: whether to call through to native.register_toolchains.
             Should be True for WORKSPACE users, but false when used under bzlmod extension
     """
-    bazel_features_deps()
-    register_jq_toolchains(register = register)
-    register_tar_toolchains(register = register)
-    register_coreutils_toolchains(register = register)
-    register_copy_to_directory_toolchains(register = register)
-    register_zstd_toolchains(register = register)
+    if register:
```

Comment:
This is needed to allow calling of `oci.toolchains` in bzlmod.

Reply:
Sure, but it seems surprising to couple that change with this added attribute? Is it just for convenience to put them in the same PR?

Reply:
this surfaced by https://github.com/bazel-contrib/rules_oci/pull/786/files#diff-913c901b3a516744550c8d29a001ca03b41ca34d06bbbf585842553367c96dd5R13 and was wrong from get go. i can probably split it into a separate PR, its just going to take more cycles for us to land this.